SCons/Environment.xml

```diff
@@ -3416,42 +3416,35 @@ env.UpdateValue(target = Value(output), source = Value(input))
 </arguments>
 <summary>
 <para>
-Sets up an alternate build location.
-When building in the <parameter>variant_dir</parameter>,
-&SCons; backfills as needed with files from <parameter>src_dir</parameter>
-to create a complete build directory.
+Sets up a mapping to define a variant build directory
+<parameter>variant_dir</parameter>,
+which must be in or underneath the project top directory.
```

Comment:
not true... it can be anywhere, but won't build by default if not under .

Reply:
Existing wording...  but I can make an edit to reflect that. So, something like this?

> `src_dir` must not be underneath `variant_dir`.  Note if `variant_dir` is not under the project top directory, target selection rules will not pick targets in the variant directory unless they are explicitly specified.

Reply:
Updated.